lisa/tools/dmesg.py

```diff
@@ -37,29 +35,32 @@ def get_output(self, force_run: bool = False) -> str:
         command_output = self._run(force_run=force_run)
         return command_output.stdout
 
-    def check_kernel_panic(
-        self, force_run: bool = False, throw_error: bool = True
+    def check_kernel_log(
```

Comment:
It looks this method is still checking panics in kernel. `check log` is too general to understand what's checked.

Reply:
I want to reuse this function to check call trace/panic.